metrics/README.md

```diff
@@ -0,0 +1,168 @@
+
+## Prometheus Configuration
+
+Due to the pull nature of Prometheus we need to configure the HTTP endpoint where it will scrape the metrics. We've already configured SPIRE to expose the HTTP endpoint via the telemetry configuration so now we need to indicate to Prometheus that it should collect metrics from that endpoint. We achieve this by configuring the target with the hostname of the SPIRE server (or SPIRE Agent) and the correct port number (e.g. 8088 for the SPIRE Server and 8089 for the SPIRE Agent).
+
+By default the HTTP resource path to fetch metrics from targets is `/metrics` but SPIRE does not expose metrics on that path. Instead, it does on the `/` path. These configurations are part of the [prometheus.yml](prometheus/prometheus.yml) configuration file.
+
+```console
+scrape_configs:
+  - job_name: 'spire-server'
+    metrics_path: '/'
+    static_configs:
+    - targets: ['spire-server:8088']
+
+  - job_name: 'spire-agent'
+    metrics_path: '/'
+    static_configs:
+    - targets: ['spire-agent:8089']
+```
+
+To run Prometheus we use the official Docker image and we mount the local directory `prometheus` to make the [prometheus.yml](/prometheus/prometheus.yml) configuration file available at the container.
+
+```console
+prometheus:
+  image: prom/prometheus:v2.20.1
+  container_name: prometheus
+  hostname: prometheus
+  restart: always
+  volumes:
+    - ./prometheus:/etc/prometheus
+  ports:
+    - "9090:9090"
+```
+
+
+## Run the Scenario
+
+Use the `set-env.sh` script to run all the services that make up the scenario. The script starts the SPIRE Server, SPIRE Agent, Graphite-Statsd and Prometheus services.
+
+Ensure that the current working directory is `.../spire-tutorials/metrics` and run:
+
+```console
+bash scripts/set-env.sh
+```
+
+Once the script is completed, in another terminal run the following command to review the logs from all the services:
+```console
+docker-compose logs -f -t
+```
+
+
+# Part 2: Test the Deployments
+
+Let's see some real data. Open the browser and navigate to `http://localhost/` to see the Graphite web UI and, on a different tab, navigate to `http://localhost:9090/` to access the Prometheus web UI.
+
+To generate some data, let's create a workload registration entry using the following script:
+
+```console
+bash scripts/create-workload-registration-entry.sh
+```
+
+And with this other script we perform requests to fetch an SVID for that new workload. These requests will serve to generate some metrics.
+
+```console
+bash scripts/fetch_svid.sh
+```
+
+Wait a couple of minutes while metrics are collected and then you can create graphs to review them.
+
+There are different metrics exported by SPIRE that can be analyzed. A complete list of them can be found on [here](https://github.com/spiffe/spire/blob/master/doc/telemetry.md). As an example, the following images show a graph for the different values of the remaining TTL of the SVID fetched.
```

Comment:
Max's PR to create that doc is still under review. We should probably remove this link for now and add it later when the PR is approved.

Please verify my change here.

```suggestion
There are different metrics exported by SPIRE that can be analyzed. A complete list of them can be found on [here](https://github.com/spiffe/spire/blob/master/doc/telemetry.md). As an example, the following images show a graph of the remaining TTL of each SVID fetched.
```

Reply:
Yeah, that is what I meant to suggest. Sorry, I should have made that change in the diff. When we add it back, you can omit "on" from the sentence: "...can be found here".

Actually, it's a little casual to use "here: for a link, although you see it all the time. A more professional approach would be: 

The [Telemetry](https://github.com/spiffe/spire/blob/master/doc/telemetry.md) page has a complete list of metrics exported by SPIRE.

Reply:
Cool, I'm taking note of that sentence to use it once the other PR is merge. Thanks.